src/compute/src/compute_state.rs

```diff
@@ -300,6 +300,7 @@ pub(crate) struct ActiveComputeState<'a, A: Allocate> {
 }
 
 /// A token that keeps a sink alive.
+#[allow(dead_code)]
 pub struct SinkToken(Box<dyn Any>);
```

Comment:
Wait, how is this dead code? It's used by `CollectionState`.

Reply:
Yes, will move these to the struct fields! These all are instances of structs where the inner field is not accessed